monifu/shared/src/main/scala/monifu/reactive/internals/operators/debounce.scala

```diff
@@ -45,7 +45,7 @@ private[reactive] object debounce {
         private[this] var hasValue = false
 
         locally {
-          scheduleNext(timeoutMillis)
+          scheduleNext(initialDelay.toMillis)
```

Comment:
Oh, this isn't so easy and won't work. If there is an `initialDelay`, it should be relative to the *last event emitted* by the source. Your `initialDelay` is relative to the start of your data source. And that is useless.